spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/sql/SqlServerJdbcDataSourcePropertiesUpdater.java

```diff
@@ -33,13 +33,11 @@ String getUserName() {
 
     @Override
     String getUrl(DataSourceProperties dataSourceProperties) {
-        SqlServer sqlServer =
-            azureAdmin.getOrCreateSqlServer(azureSqlProperties.getServerName(), dataSourceProperties.getUsername(),
-                    dataSourceProperties.getPassword());
-        azureAdmin.createSqlDatabaseIfNotExists(azureSqlProperties.getServerName(),
+        SqlServer sqlServer = azureAdmin
+                .getOrCreateSqlServer(azureSqlProperties.getServerName(), dataSourceProperties.getUsername(),
+                        dataSourceProperties.getPassword());
+        azureAdmin.getOrCreateSqlDatabase(azureSqlProperties.getServerName(), azureSqlProperties.getDatabaseName());
+        return String.format(databaseType.getJdbcUrlTemplate(), sqlServer.fullyQualifiedDomainName(),
```

Comment:
could you please add some local variable to hold something like `azureSqlProperties.getServerName()` ? From my side, wrap lines is difficult to read.